<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>671</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>937</width>
    <height>671</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>937</width>
    <height>671</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow &gt; QWidget{
   
	background-image: url(:/Icons/Resources/css-pattern-by-magicpattern.png);
	background-color: (39,39,39);
	background-position: center;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>641</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>641</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>641</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-color: rgb(167, 209, 41);
border-width: 2px;
border-radius: 10px;
border-style: outset;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Panton-Trial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(226, 220, 200);
border-color: rgb(167, 209, 41);
border-width: 2px;
border-radius: 10px;
border-style: outset;
background-color: #3E432E;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Panton-Trial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(226, 220, 200);
border-color: rgb(167, 209, 41);
border-width: 2px;
border-radius: 10px;
border-style: outset;
background-color: #3E432E;</string>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Panton-Trial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(226, 220, 200);
border-color: rgb(167, 209, 41);
border-width: 2px;
border-radius: 10px;
border-style: outset;
background-color: #3E432E;</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Panton-Trial</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(226, 220, 200);
border-color: rgb(167, 209, 41);
border-width: 2px;
border-radius: 10px;
border-style: outset;
background-color: #3E432E;</string>
       </property>
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>761</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 220, 200);
border-color: rgb(167, 209, 41);
border-width: 2px;
border-radius: 10px;
border-style: outset;</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>641</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>761</width>
       <height>641</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>761</width>
       <height>641</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(226, 220, 200);
background-color: rgb(39, 39, 39);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
